package.json

```diff
@@ -23,7 +23,7 @@
     "method-override": "2.3.3",
     "moment": "2.10.3",
     "passport": "0.2.2",
-    "passport-saml": "0.5.3",
```

Comment:
Not readily available, but there is one in the Engineering AWS account. I'll spin it up and give this a spin later this week (you may need to prod me if I forget).